ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -1366,6 +1366,7 @@ private void fetchRequests(RequestRegistrationCallback<PutOperation> requestRegi
               replicaId.getDataNodeId());
         }
         routerMetrics.getDataNodeBasedMetrics(replicaId.getDataNodeId()).putRequestRate.mark();
+        routerMetrics.routerPutRequestRate.mark();
```

Comment:
We have node-level `putRequestRate` and `getRequestRate`, can we aggregate them without introducing new metrics?

Reply:
Talked with @jsjtzyy  offline. We will add this metric this time because aggregate is a little bit complex.